api/Worker.json

```
@@ -6,7 +6,8 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-worker-interface",
         "support": {
           "chrome": {
-            "version_added": "2"
+            "version_added": "2",
+            "notes": "Not available in Shared Workers. See <a href='https://crbug.com/40695450'>bug 40695450</a>."
```

Let's remove this note, as it duplicates the information below:

```suggestion
            "version_added": "2"
```